scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -449,11 +450,12 @@ public void getStoredLogs(AppenderProvider logSink) {
         taskLogger.getStoredLogs(logSink);
     }
 
+    public KeyPair getKeyPair() throws NoSuchAlgorithmException {
```

Comment:
I don't see the need of creating a new method here.

Reply:
To make TaskLauncherTest to pass